### PR TITLE
feat: category component

### PR DESCRIPTION
**Pull request by alexzavgorodnii**
**Name**
feat: category component

**Purpose of pull request**
Added Category component 

**Additional information**
image
image

**Ticket**
#26

### CODE REVIEW

File: public/locales/uk/translation.json [hunk 1]
```diff
@@ -35,5 +35,10 @@
   },
   "searchBar": {
     "PLACEHOLDER": "що вас цiкавить?"
-  }
+  },
+  "ALL_PROJECTS": "Всі проекти",
```
Comment (braindrain9) [inline public/locales/uk/translation.json:39]: alphabetical order please

File: public/locales/uk/translation.json [hunk 2]
```diff
@@ -35,5 +35,10 @@
...
+  "COVID": "COVID-19",
+  "PROJECTS_0": "{{count}} проект",
```
Comment (braindrain9) [inline public/locales/uk/translation.json:41]: the same here

File: public/locales/uk/translation.json (hunk 1 above)
Comment (alexzavgorodnii) [reply public/locales/uk/translation.json]: Done

File: public/locales/uk/translation.json (hunk 2 above)
Comment (alexzavgorodnii) [reply public/locales/uk/translation.json]: Done

File: .github/workflows/pipeline_master.yml [hunk 3]
```diff
@@ -1,4 +1,4 @@
-name: Staging web publishment. 
+name: Staging web publishment.
```
Comment (braindrain9) [inline .github/workflows/pipeline_master.yml:1]: please remove unrelated changes

File: README.md [hunk 4]
```diff
@@ -4,7 +4,6 @@ Master `[master branch]` latest build is available on https://staging.mecenat.co
 
 Development `[develop branch]` latest build is available on https://development.mecenat.com.ua/
 
-
 ## Available Scripts
```
Comment (braindrain9) [inline README.md:7]: the same here

Comment (alexzavgorodnii) [reply README.md:7]: done

File: .github/workflows/pipeline_master.yml (hunk 3 above)
Comment (alexzavgorodnii) [reply .github/workflows/pipeline_master.yml]: done

File: src/shared/components/Category/component.tsx [hunk 5]
```diff
@@ -0,0 +1,22 @@
+import React from 'react';
+import { CategoryWrapper, TitleWrapper, CountWrapper, IconWrapper } from './styles';
+
+type CategoryProps = {
+  title: string;
+  count?: string;
+  icon: string;
```
Comment (braindrain9) [inline src/shared/components/Category/component.tsx:7]: we should add some `link` prop here, and pass it down, to make one category card to be a link.

File: src/app/pages/HomePage/Categories/component.tsx [hunk 6]
```diff
@@ -0,0 +1,33 @@
...
+import Category from 'shared/components/Category';
+import { AllProjectsIcon, ArtIcon, ChildrenIcon, CovidIcon, EcoIcon, EducationIcon } from 'shared/images';
+
+const Categories: React.FC = () => {
+  const { t } = useTranslation();
+
+  return (
+    <Row>
+      <Col md={4}>
+        <Category title={t('EDUCATION')} count={t('PROJECTS', { count: 43 })} icon={EducationIcon} />
```
Comment (braindrain9) [inline src/app/pages/HomePage/Categories/component.tsx:13]: please, keep these values in `categories` array, like `menuItems`, and then map items

```js
categories.map(category => {
 <Col md={4}>
     <Category 
        title={category.title} 
        count={t('PROJECTS', { count: category.count })} 
        icon={category.icon} 
        link={category.link}
      />
  </Col>
})

```

Comment (alexzavgorodnii) [reply src/app/pages/HomePage/Categories/component.tsx]: Done

File: src/shared/components/Category/component.tsx (hunk 5 above)
Comment (alexzavgorodnii) [reply src/shared/components/Category/component.tsx:6]: done

File: src/app/pages/HomePage/Categories/component.tsx [hunk 7]
```diff
@@ -0,0 +1,57 @@
...
+    {
+      title: t('ALL_PROJECTS'),
+      icon: AllProjectsIcon,
+      link: '/',
+    },
+  ];
+  return (
+    <Row>
+      {categoriesList.map((category: CategoryProps, index) => (
+        <Col md={4} key={index}>
```
Comment (braindrain9) [inline src/app/pages/HomePage/Categories/component.tsx:50]: let's add some `id: Math.random()` and put it instead of `key={index}`

Comment (alexzavgorodnii) [reply src/app/pages/HomePage/Categories/component.tsx]: done

**Comment by braindrain9** [comment]
please merge `master` into `develop` branch after merging this PR.

**Comment by braindrain9** [comment]
also - next time please clean-up the description in PR commit to avoid this

Screenshot 2020-05-10 at 14 57 24
